atd-vze/src/views/Users/User.js

```diff
@@ -67,17 +67,17 @@ const User = () => {
       const label = userAttributes[key].label;
       let formattedValue = null;
 
-      const format = userAttributes[key].format;
+      const attributeFormat = userAttributes[key].format;
 
-      switch (format) {
+      switch (attributeFormat) {
         case "string":
           formattedValue = user[key];
           break;
         case "bool":
           formattedValue = !!user[key] ? "Yes" : "No";
           break;
         case "time":
-          formattedValue = moment(user[key]).format("MM/DD/YYYY, h:mm:ss a");
+          formattedValue = format(parseISO(user[key]), "MM/dd/yyyy, h:mm:ss a");
```

Comment:
This is another instance where I couldn't test if my change worked because there's a bug in VZE where if I click on a user from the Users tab, the page just loads forever. Once again I'm assuming `user[key]` is a string, I'd like to confirm this but I'm not sure how 🤔

Reply:
Update:  @frankhereford checked and it looks right on his end. Still need to figure out why these pages don't load for me, could have something to do with permissions even though I do have admin permissions?

Reply:
Looks great to me as well 👍 Also noticed that the AM is now capitalized just like it is elsewhere in the app so that is a nice improvement.
<img width="323" alt="Screen Shot 2022-10-11 at 11 56 39 AM" src="https://user-images.githubusercontent.com/37249039/195165333-2208072b-2e71-4286-bdf7-8a3d8251aa5e.png">
<img width="323" alt="Screen Shot 2022-10-11 at 11 56 33 AM" src="https://user-images.githubusercontent.com/37249039/195165343-722d1561-4545-4786-894b-f06b3c9a4f88.png">